src/rebar_fetch.erl

```diff
@@ -61,7 +61,7 @@ download_source(AppDir, Source, State) ->
     catch
         C:T ->
             ?DEBUG("rebar_fetch exception ~p ~p ~p", [C, T, erlang:get_stacktrace()]),
-            throw(?PRV_ERROR({fetch_fail, Source}))
+            ?PRV_ERROR({fetch_fail, Source})
```

Comment:
That's a pretty major change there. How do you know that's safe? This throw probably means that it should interrupt whatever was going on, not just return a value we can ignore.